codegens/swift/test/unit/fixtures/testcollection/collection.json

```diff
@@ -721,11 +721,11 @@
 					"raw": "Duis posuere augue vel cursus pharetra. In luctus a ex nec pretium. Praesent neque quam, tincidunt nec leo eget, rutrum vehicula magna.\nMaecenas consequat elementum elit, id semper sem tristique et. Integer pulvinar enim quis consectetur interdum volutpat."
 				},
 				"url": {
-					"raw": "https://mockbin.org/request/:action",
+					"raw": "https://postman-echo.com/request/:action",
```

Comment:
`https://postman-echo.com/request/:action` gives 404. Is this correct?

Reply:
@VShingala It is a path variable 
the complete URL is like this

```
"url": {
	"raw": "https://postman-echo.com/request/:action",
	"protocol": "https",
	"host": [
		"postman-echo",
		"com"
	],
	"path": [
		"request",
		":action"
	],
	"variable": [
		{
			"key": "action",
			"value": "post"
		}
	]
},
```

Reply:
@aman-v-singh Even with replacing value of `:action` to `post`, the request to the corresponding URL returns 404.

Reply:
@VShingala Fixed it by changing it to `https://postman-echo.com/:action`